testCommon/src/test/resources/common.sh

```diff
@@ -50,9 +65,10 @@ cmt() {
   touch $f
   git add $f
   git commit -m "$*"
+  set_fake_git_date 2020-01-$((++commit_day_of_month))
```

Comment:
Reflog in machete-sandbox now looks like:
![image](https://user-images.githubusercontent.com/3383210/86397975-208da180-bca5-11ea-8e8c-026f0492ac76.png)

Reply:
(2020-01-01 to 2020-01-12)

Reply:
Not sure but maybe `commit_day_of_month` may be a argument of `commit()` and this can be more flexible then?

Reply:
Hmm TBH I don't think I want it particularly flexible... as for now just a simple 1-day-per-commit timeline should be enough